.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Daily might be a little extreme, is there a weekly one?

Reply:
Yeah there is a weekly one :D I will change it to that

Reply:
Now changed to check weekly :D